### PR TITLE
Removed module identity on package removal

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

* Removed osconfig module identity on Debian/RPM package removal

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] I added unit-tests to validate my changes. All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

File: devops/debian/prerm [hunk 1]
```diff
@@ -44,4 +44,9 @@ if [ -d /run/systemd/system ] && [ "$1" = remove ]; then
 fi
 # End automatically added section
 
+# MSFT change: Remove module identity on package removal
```
Comment (MariusNi) [inline devops/debian/prerm:47]: >MSFT [](http://example.com/codeflow?start=2&length=4)

Do we need "MSFT" in comments? MSFT is an internal term that should not appear in our public repo.

We can use "OSConfig Core Team" if needed.

Comment (AhmedBM) [reply devops/debian/prerm:47]: This was the pattern we used for any of the changes made after applying the debhelper templates, its not the only place with the "MSFT changes", they are also defined in https://github.com/Azure/azure-osconfig/blob/main/devops/debian/postinst

If we change, we should change all instances.

Comment (MariusNi) [reply devops/debian/prerm:47]: Yes, please lets' change all comments and names stating "MSFT" which are not really appropriate for our public repro. Instead use "Core Team" or "OsConfig Core Team" as referenced in out documents in this repo. Thanks!

**Review by MariusNi — APPROVED** [review]
:shipit:

**Pull request merged**